### DISCUSSION TITLE
2일차 Node 프로그래밍 강의자료 17쪽 질문입니다.

### DISCUSSION THREAD

**Discussion by ahystroy** [Road-Balance/gcamp_ros2_basic#43] [Q&A]
node.get_logger()로 출력하면 [INFO]로 시작하는 Logger가 출력되어야하는데, 실행시키면 뒤에 로거 정보는 나오지 않고 다음과 같이 출력됩니다.

[INFO]: 
==== Hello ROS 2 ====

[INFO] 뒤에 나올 [시간]과 [노드 이름] 정보가 출력되지 않습니다. 수업을 못따라와서 다시 복습하는 중입니다. 혹시 코드에서 더 추가해야할 코드가 있을까요?

**Comment by kimsooyoung** [comment]
해당 소스 코드를 한번 공유주실 수 있을까요?

**Comment by ahystroy** [comment]
git clone해서 받아온 소스코드 그대로입니다.

2022년 7월 14일 (목) 오전 6:30, Soo Young Kim (Swimming Kim) <
***@***.***>님이 작성:

> 해당 소스 코드를 한번 공유주실 수 있을까요?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Road-Balance/gcamp_ros2_basic/discussions/43#discussioncomment-3141200>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2BKGEV2K7MJU27T5YF5EL3VT4YQHANCNFSM53OXURNQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>

-- 
한국산업기술대학교가 한국공학대학교로 새롭게 도약합니다.

KPU changes its name to TU Korea(Tech 
University of Korea)

(우리 대학의 도메인이 kpu.ac.kr <http://kpu.ac.kr/> -> 
tukorea.ac.kr <http://tukorea.ac.kr/>로 변경됩니다)

(New domain name is 
tukorea.ac.kr <http://tukorea.ac.kr/>)

**Comment by kimsooyoung** [comment]
동일 코드 사용 시 제 환경에서는 문제가 없습니다. 

![image](https://user-images.githubusercontent.com/12381733/179088694-919e2a8f-80d1-40e6-bafa-b5a93fbfdf8b.png)

상세한 정보를 주셔야 제대로 된 답변이 가능합니다.
